### DISCUSSION TITLE
Paper Component - Elevation like MUI

### DISCUSSION THREAD

**Discussion by JavaScriptJohn** [mantinedev/mantine#2233] [Feature requests]
It would be really nice to have a way to easily separate the background color/box shadow of a paper/card component. This really helps when nesting in certain areas. MUI did a really good job with it, their implementation is pretty complicated but works well. Is this something that you would consider?

`<Paper elevation={0} />`
`<Paper />`
`<Paper elevation={3} />`

![image](https://user-images.githubusercontent.com/849558/186629156-efac2550-ea15-4744-8d4e-dd67cbd207ee.png)

https://mui.com/material-ui/react-paper/#elevation

**Comment by zhulien-ivanov** [comment]
This is actually a very good idea. I've also encountered a couple of cases where this can be beneficiary being supported on a library level.

**Comment by JavaScriptJohn** [comment]
@rtivital Thoughts on this?

**Reply by rtivital** [reply]
I think it's fine, though I do not have any theme tokens that can work as intended for dark theme.
